sw/device/tests/BUILD

```diff
@@ -1475,6 +1475,46 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "entropy_src_bypass_mode_health_test",
+    srcs = ["entropy_src_bypass_mode_health_test.c"],
+    exec_env = dicts.add(
+        EARLGREY_TEST_ENVS,
+        EARLGREY_SILICON_OWNER_ROM_EXT_ENVS,
+        {
+            "//hw/top_earlgrey:fpga_cw310_sival": None,
```

Comment:
I would drop this exec_env to avoid running this test twice on the SiVal nightlies.